Exec/mhd_tests/OrszagTang/problem_initialize_mhd_data.H

```diff
@@ -22,7 +20,7 @@ void problem_initialize_mhd_data (int i, int j, int k,
 
     Real z = 0.0;
 #if AMREX_SPACEDIM == 3
-    z = problo[2] + dx[2] * (static_cast<Real>(k) + 0.5_rt);
+    //z = problo[2] + dx[2] * (static_cast<Real>(k) + 0.5_rt);
```

Comment:
Any reason for commenting these instead of deleting?

Reply:
no reason.  I'll delete them